configure.ac

```diff
@@ -512,23 +512,29 @@ EXTRA_WARNINGS=""
 
 WARNLIST="
 	all
+	extra
```

Comment:
extra includes several warning types that significantly increase the ability for the compiler to warn about potential pitfalls.